driver/bpf/plumbing_helpers.h

```diff
@@ -634,6 +725,8 @@ static __always_inline void call_filler(void *ctx,
 
 	++state->n_evts;
 
+	state->tail_ctx.socketcall_syscall_id = socketcall_syscall_id;
```

Comment:
Store this in our `tail_ctx` so that it is made available to `bpf_syscall_get_argument`.